deployment/docker/docker-compose.yml

```diff
@@ -113,6 +113,10 @@ services:
       - KEYCLOAK_ENABLE_CLIENT_AUTH=${KEYCLOAK_ENABLE_CLIENT_AUTH:-false}
       - KEYCLOAK_WEB_CLIENTID=${KEYCLOAK_WEB_CLIENTID:-forms-flow-web}
       - FORMSFLOW_ADMIN_URL=${FORMSFLOW_ADMIN_URL:-}
+      - REDIS_HOST=${REDIS_HOST}
```

Comment:
Can you please add a default value for REDIS_HOST, to not cause a warning ?

Reply:
updated